frontend/src/components/Vote.js

```diff
@@ -133,39 +146,34 @@ export default function VoteDialog(props) {
                   }
                   label="Abstain"
                 />
-
               </FormGroup>
             </FormControl>
-            {
-              props.isAuthzMode && props.granters.length > 0 ?
-                <FormControl fullWidth
-                  sx={{
-                    mt: 1,
+            {props.isAuthzMode && props.granters.length > 0 ? (
```

Comment:
no need to check length > 0, whenever we need to check length. (props.isAuthzMode && props.granters.length) is enough.